python/ray/util/spark/cluster_init.py

```diff
@@ -935,6 +937,9 @@ def setup_ray_cluster(
             or referenced objects (either in-memory or spilled to disk). This parameter
             does not affect the head node.
             Default value is 1.0, minimum value is 0
+        autoscale_min_worker_nodes: Set minimal number of worker nodes, this argument
```

Comment:
```
If autoscale=True, then the ray cluster starts with zero worker node,
```

line 856 is wrong now

Reply:
It's weird that we have `num_worker_nodes` and `autoscale_min_worker_nodes`. I think we should unify autoscaling cluster and static cluster. static cluster is just a autoscaling cluster with same min and max.

Probably we should rename `num_worker_nodes` to `max_worker_nodes` and `autoscale_min_worker_nodes` to `min_worker_nodes`.

Reply:
OK I can deprecate `num_worker_nodes`. But I don't think we can brutely remove the argument in next version, it is breaking change and this is frequently used argument.